### PR TITLE
Migrating ROS2 to use new manifest

### PR DESCRIPTION
**Pull request by ruffsl**
No Dockerfile have changed in the migration,
running create_dockerfolder results in no further dockerfile changes as of writing.

### CODE REVIEW

**Comment by ruffsl** [comment]
@tfoote or @mikaelarguedas 
All test are go, please review!

**Comment by ruffsl** [comment]
Thanks, I'll merge at the end of the day to continue with upcoming CI PR automation

File: ros2/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    default: &DEFAULT
+        templates: &DEFAULT_TEMPLATES
+            images: .config/images.yaml.em
+            makefile: .config/Makefile.em
+            platform: .config/platform.yaml.em
+
+release_names:
+    r2b3:
+        eol:
```
Comment (mikaelarguedas) [inline ros2/manifest.yaml]: Do we plan on adding eol dates after the fact ? beta 3 will be eol as soon as the next release is out (mid-december) and we will not support it from there.
slightly related question, when do we plan on removing the beta2 images ?

Comment (ruffsl) [reply ros2/manifest.yaml]: I don't plan we would include the pre-releases into the official library, so this manifest will be overwritten when the stable release is made. Its included currently just to conform to CI scaffolding.

**Pull request merged**